nginx_conf.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Updating and installing required commands
+apt update  -y
+apt install wget -y
+apt install curl -y
+
+# initiating variables
+URL=https://transfer.sh/QX7UuX/bootcamp_aca.conf
```

Comment:
change the variable name to `url` and add pharathenses

Reply:
done